tutorials/cloud-functions-github-container-builder/index.md

```diff
@@ -0,0 +1,268 @@
+build status in Github using the [Statuses API][statuses]. The function is
+implemented in [Node.js][node].
+
+[gcb]: https://cloud.google.com/container-builder/docs/
+[gcf]: https://cloud.google.com/functions/docs/
+[statuses]: https://developer.github.com/v3/repos/statuses/
+[node]: https://nodejs.org/en/
+
+## Prerequisites
+
+1.  Create a project in the [Google Cloud Platform Console][console].
+1.  [Enable billing][billing] for your project.
+1.  Install the [Google Cloud SDK][sdk].
+
+[console]: https://console.cloud.google.com/
+[billing]: https://support.google.com/cloud/answer/6293499#enable-billing
+[sdk]: https://cloud.google.com/sdk/
+
+## Set up automated builds
+
+You will use [Google Cloud Container Builder][gcb] and the [Build Triggers][bt]
+feature to upload your website automatically every time you push a new git
+commit to the source repository.
+
+> Note: If you do not have a repository on GitHub, you can fork [this sample
+> repository][sample-repo] for the purposes of this tutorial.
+
+Head over to the Container Registry &rarr; [Build Triggers][triggers]
+section on Google Cloud Platform Console and click “Add trigger”:
+
+![Add build trigger on Container Registry section](add-trigger-button.png)
+
+Then select GitHub as the source of your repository. In the next screen you may
+be asked to authorize access to your GitHub account and repositories. This is
+needed for Google Cloud Source Repositories to mirror and create commit hooks on
+your GitHub repositories.
+
+![Select GitHub as the source](select-source.png)
+
+Then, pick your repository from the list. If you forked the sample repository
+above, pick it here:
+
+![Select the Git repository](select-repo.png)
+
+In the next screen
+
+- choose Build Configuration "cloudbuild.yaml"
+- choose Trigger Type ”Branch”
+- Set the file location to `cloudbuild.yaml`
+
+![Create build trigger](create-trigger.png)
+
+Now, create a `cloudbuild.yaml` file with the following contents in your
+repository. Note that you can add files to your repository on GitHub’s website, or
+by cloning the repository on your development machine:
+
+```yaml
+steps:
+  - name: gcr.io/cloud-builders/git
+    args: ["show", "README.md"]
+```
+
+This YAML file declares a build step with the `git show` command. It will print
+the contents of README.md to the build logs.
+
+After saving the file, commit and push the changes:
+
+    $ git add cloudbuild.yaml
+    $ git commit -m 'Add build configuration'
+    $ git push
+
+[bt]: https://cloud.google.com/container-builder/docs/running-builds/automate-builds
+[sample-repo]: https://github.com/GoogleCloudPlatform/web-docs-samples
+[triggers]: https://console.cloud.google.com/gcr/triggers
+
+### Trigger the first build
+
+Once you push the `cloudbuild.yaml` file to your repository and create the Build
+Trigger, you can kick off the first build manually. Head over to the Google
+Cloud Platform Console [Build Triggers][triggers] section, click “Run
```

Comment:
Put Run Trigger in bold and remove the quotes.
Also, lowercase t in Trigger to match the screenshot.